app/src/main/java/app/revanced/manager/patcher/ProgressEventParcel.kt

```diff
@@ -0,0 +1,105 @@
+package app.revanced.manager.patcher
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+
+@Parcelize
+data class ProgressEventParcel(
```

Comment:
Why is this type necessary? The types in `PatcherProgress.kt` are already parcelable.

Reply:
Because AIDL doesn’t support Kotlin sealed classes

Reply:
Fair, but wouldn't this work then?:

```kt
@Parcelize
data class ProgressEventParcel(val event: ProgressEvent) : Parcelable
```
If it doesn't, use a `Bundle` and `Bundle.getParcelable`.